app/config.go

```diff
@@ -0,0 +1,61 @@
+package app
```

Comment:
I think it would be better to put this commit in the next pr.
This is because new features have already been approvaed by other reviewers before they are uploaded.